src/components/CommentForm/CommentForm.module.css

```diff
@@ -0,0 +1,41 @@
+.FormActions {
```

Comment:
Maybe? https://github.com/taskany-inc/bricks/issues/813

Reply:
It would be better in a separate PR